justfile

```diff
@@ -101,15 +101,9 @@ py-build *ARGS:
 py-format:
     #!/usr/bin/env bash
     set -euo pipefail
-    # The order below is important and sadly we need to call black twice. Ruff does not yet
-    # fix line-length (See: https://github.com/astral-sh/ruff/issues/1904).
-    #
-    # 1) Call black, which among others things fixes line-length
-    # 2) Call ruff, which requires line-lengths to be correct
-    # 3) Call black again to cleanup some whitespace issues ruff might introduce
-    black --config rerun_py/pyproject.toml {{py_folders}}
+    # NOTE: the order here matches the one in `crates/re_types_builder/src/format/python.rs`:
     ruff --fix --config rerun_py/pyproject.toml {{py_folders}}
-    black --config rerun_py/pyproject.toml {{py_folders}}
+    ruff format --config rerun_py/pyproject.toml {{py_folders}}
```

Comment:
is there a deeper story that this isn't a pixi task yet? There is a pixi task for checking but not for formatting 🤔

Reply:
I mean, this is the deeper story: https://github.com/rerun-io/rerun/issues/3717 :)

Reply:
There is also the `{[py_folders}}` which I don't know if `pixi` supports. Anyway, this PR is not about moving more tasks into pixi